experiments/fiction_experiments.hpp

```diff
@@ -49,15 +49,61 @@ inline constexpr uint64_t xor5_maj         = 0b000000000000000000000000000000000
 inline constexpr uint64_t parity           = 0b0000000000000000000000000000000000000000010000000000000000000000;
 inline constexpr uint64_t fontes18         = 0b0000000000000000000000000000000000000000011111111111111110000000;
 
```

Comment:
We can take the original "fiction_experiments.hpp" version from you. Was just an experiment a few weeks ago.